docs/snippets/license_preamble.txt

```diff
@@ -0,0 +1,20 @@
+This file is part of %package_name%.
+
+Developed for the LSST Data Management System.
+This product includes software developed by the LSST Project
+(http://www.lsst.org).
+See the COPYRIGHT file at the top-level directory of this distribution
+for details of code ownership.
+
+This program is free software: you can redistribute it and/or modify
```

Comment:
GNU recommends we replace "This program" with the actual package name in the preamble. I'm happy to do that, especially if the templates builder can fill it in.